oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/prometheus/rule/Rules.java

```diff
@@ -47,29 +50,72 @@ public static List<Rule> loadRules(final String path) throws ModuleStartExceptio
     public static List<Rule> loadRules(final String path, List<String> enabledRules) throws ModuleStartException {
         File[] rules;
         try {
-            rules = ResourceUtils.getPathFiles(path);
+            rules = ResourceUtils.getAllPathFiles(path);
         } catch (FileNotFoundException e) {
             throw new ModuleStartException("Load fetcher rules failed", e);
         }
+        final List<String> formedEnabledRules =
```

Comment:
Could you build UTs for the new logic? Such as through `loadRules` method?

Reply:
Got it.

Reply:
> Could you build UTs for the new logic? Such as through `loadRules` method?

What does `build UTs` mean? Is it extract a method?

Reply:
Unit Tests. The path/file pattern mechanism is added newly, and it is better if you could test through correct input and wrong put with predictable results(correct file list or exception or empty list)

Reply:
I understand.